Machine_Learning/Bank Loan Approval Prediction/Web App/app.py

```
@@ -0,0 +1,147 @@
+                alert_message = "Your age is not within the expected range for a Masters degree."
+        elif education == 3:  # Advanced Degree
+            if age < 23:
+                alert_message = "Your age is not within the expected range for an Advanced degree."
+        
+        if alert_message:
+            return render_template('index.html', education_levels=education_levels, zip_codes=zip_codes, alert_message=alert_message)
+        
+        # Retrieve form data
+        # set it to boolean values for predictions
+        securityAcc = int(request.form['securityAccount'])
+        securityAcc = True if securityAcc == 1 else False
```

Ruff (0.8.2)

Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)